src/content/docs/en/guides/astro-db.mdx

```diff
@@ -206,17 +384,17 @@ This example inserts a row into a `Comment` table based on a parsed form POST re
 import { db, Comment } from 'astro:db';
 
 if (Astro.request.method === 'POST') {
-  // parse form data
+  // Parse form data
   const formData = await Astro.request.formData();
   const author = formData.get('author');
   const content = formData.get('content');
   if (typeof author === 'string' && typeof content === 'string') {
-    // insert form data into the Comment table
+    // Insert form data into the Comment table
     await db.insert(Comment).values({ author, content });
   }
 }
 
-// render the new list of comments on each request
+// Render the new list of comments on each request
 const comments = await db.select().from(Comment);
 ---
 
```

Comment:
The rest of the page uses body for the comments while these few examples use content

I suggest making everything uniform

```suggestion
<label for="body">Body</label>
<textarea id="body" name="body"></textarea>
```